### DISCUSSION TITLE
Install

### DISCUSSION THREAD

**Discussion by dicHedd** [Kuronons/FZ_graphics#2] [Q&A] [answered]
Is there any tutorials that are more up to date? I tried to use the "How2Flipper" pdf but the folders shown on the tutorial either in the zip or in the flipper are not there.

**Comment by Kuronons** [comment]
Hey,
Yeah possible there are some outdated info. What are you trying to do ? Basically, the tuto from Wr3nch was really to get people to know how to get files from a GH. You can reach him on many Flipper discord servers. He s mod on official one.
I haven t updated my Flip for like 4 months or so and I know there has been changes on official fw, so maybe that's it... 
But for just animations, nothing complicated. A bit more for profile/passport picts. And it depends on which fw you re on too.

**Comment by bad-antics** [comment, marked as answer]
Hey! The process is actually simpler than the PDF makes it seem. Here's a quick guide:

### Installing Flipper Zero Graphics/Animations

**Method 1: Using qFlipper (Easiest)**
1. Connect your Flipper to your computer
2. Open qFlipper
3. Go to the **File Manager** tab
4. Navigate to `SD Card/dolphin/` 
5. Create a folder for your animation pack if it doesn't exist
6. Drag and drop the animation folders into the appropriate location

**Method 2: Direct SD Card**
1. Remove the SD card from your Flipper
2. Insert it into your computer
3. Navigate to the `dolphin/` folder
4. Copy animation folders there
5. Edit `manifest.txt` in the dolphin folder to register the new animations

**Method 3: Using Momentum/Unleashed Firmware**
Custom firmwares like Momentum have a built-in asset pack system:
1. Download the asset pack (`.zip` file)
2. Place it in `SD Card/asset_packs/`
3. Go to Settings > Desktop > Animations on your Flipper
4. Select the asset pack

### Folder Structure
The animations need to follow this structure:
```
SD Card/
  dolphin/
    YourAnimation/
      frame_0.bm
      frame_1.bm
      ...
      meta.txt
```

The `meta.txt` file defines the animation properties (dimensions, frame rate, frame order).

### For Kuronons' graphics specifically:
Download the release ZIP from the repo, extract it, and copy the folders to your SD card's `dolphin/` directory. The animations should be pre-formatted with the correct folder structure and meta.txt files.

**Reply by Kuronons** [reply]
Hi,

Yes, nothing complicated. You may just want to specify that the **folder structure** is different on CFW using asset packs such as Momentum as it shows in **Method 3**.

```
SD/ (or /ext in File Browser)
    |-asset_packs/
        |-PackName/
            |-Anims/
                |-ExampleAnim1/
                    |-frame_0.bm
                    |-frame_1.bm
                    |...
                    |-meta.txt
                |-ExampleAnim2/
                    |...
                |-manifest.txt
            |-Icons/
                |Passport/
                     |-passport_128x64.bmx
                     |-passport_bad_46x49.bmx
                     |-passport_happy_46x49.bmx
                     |-passport_okay_46x49.bmx
                |...
```
Specific guide for asset packs on ***Momentum wiki*** [there](https://momentum-fw.dev/wiki/)
Also Momentum provides a bunch of asset packs that can be installed directly via [the dedicated page](https://momentum-fw.dev/asset-packs) of their website (on browsers that take in charge serial com webtools)

In addition, one can want to read the guide I wrote on ***Flipper Unofficial wiki*** that explains meta.txt functions in details [there](https://flipper.wiki/tutorials/Animation_guide_meta/Meta_settings_guide/)
